packages/table/src/Table.tsx

```diff
@@ -22,37 +22,75 @@ import { TableSortOption } from './types';
 import TableContent from './TableContent';
 
 type HeaderProps = {
+  /** This makes the column sticky to the table. This works best when inside a scrollable container. Not supported in IE11. */
   sticky: boolean;
 } & React.HTMLAttributes<HTMLElement>;
 
 export type CommonTableProps<T extends IdType> = {
+  /** This is a unique id that is prepended to the table and nested table elements. **/
```

Comment:
Not the biggest deal, but documentation comments are /** */, it looks like you're adding an extra asterisk at the end which will end up adding an asterisk to the description text.